pillar/heroku/xpro.sls

```diff
@@ -0,0 +1,38 @@
+{% set app_name = 'xpro' %}
+{% set environments = ['ci', 'rc', 'production'] %}
+{% set status_token = salt.random.get_str(24) %}
+
+proxy:
+  proxytype: heroku
+
+heroku:
+  {% for env in environments %}
+  app_name: {{ app_name }}-{{ env }}
+  api_key: __vault__::secret-operations/global/heroku-api-key>data>value
+  config_vars:
+    AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/xpro-{{ env }}>data>access_key
+    AWS_SECRET_ACCESS_KEY: __vault__:cache:aws-mitx/creds/xpro-{{ env }}>data>secret_key
+    AWS_STORAGE_BUCKET_NAME: 'xpro-{{ env }}'
+    MAILGUN_URL: 'https://api.mailgun.net/v3/xpro-{{ env }}-mail.odl.mit.edu'
+    MAILGUN_KEY: __vault__::secret-operations/global/mailgun-api-key>data>value
+    MAILGUN_FROM_EMAIL: 'MIT xPRO <no-reply@xpro-{{ env }}-mail.odl.mit.edu>'
+    MITXPRO_ADMIN_EMAIL: 'cuddle-bunnies@mit.edu'
+    MITXPRO_DB_CONN_MAX_AGE: 0
+    MITXPRO_DB_DISABLE_SSL: True
```

Comment:
Probably worth adding a comment to describe the reasoning for this as I described on slack